### ISSUE TITLE
A few suggestions for the next update.

### ISSUE THREAD

**Issue by MrPanica** [Scags/TF2-Jailbreak-Redux#62]
Make it so that when displaying one annotation the other is not removed.

Make notifications about ending a rebel.

Make a tag in the chat for the warden.

Make a command for the warden, with which he could invite the Reds to the blue team.

More bosses!

If the spy has a rebel and goes into stealth, then the rebel effect also becomes invisible.

**Comment by Scags** [comment]
1. When you mention annotations, do you mean the repeat ones or the warden markers? I'm not sure if I can even do anything about it.
2. Sure.
3. Are you talking about a title or something in front of their name? IE:
```
[Warden] Scag: All reds do a 360
```
Because if that's the case, there may be some conflicting plugins with chat processors or title plugins 
or store plugins as well.

4. I guess I could do that. There may be some issue with spawning but I'll see what can be done.
5. I'd love to but that would be an issue with servers/players downloading custom models, sounds, etc. It'd be a mess. I'm tempted to move the sub-plugin last requests to an separate
6. Sure I'll take care of that.

**Comment by MrPanica** [comment]
Yes, I'm talking about the markers and annotations from the TF2JailRedux_RepeatAnnotation plugin. When warden puts a marker, and the red asks for a repeat order, the marker will disappear. Also, if one red asks for a repetition, and the other too, then one of the annotations will disappear. I saw a server where this bug is fixed.
But it seems your plugin is not installed there. But there is the same functionality (markers and annotations of repetition).

Yes you are right. I am talking about this.
Just need some kind of identifier for the warden. But this is not a high priority.

**Comment by Scags** [comment]
I'll do some research on the annotation issue. I'll also see what I can tweak with Warden chat tags.

**Comment by RWolfyo** [comment]
About Warden chat tag I found this lately and started to use it for group tags I bet you could integrate that with tf2jail redux https://forums.alliedmods.net/showthread.php?t=303671
it runs on https://forums.alliedmods.net/showthread.php?p=2448733

**Comment by Scags** [comment]
Alrighty with the latest update I've got everything but the warden tags. That's next. Just let me know if the invite command works or if there's anything wrong with it. 👍
